client/src/main/scala/it/cwmp/client/controller/messages/RoomsCommon.scala

```diff
@@ -62,6 +62,11 @@ object RoomsRequests {
     */
   sealed case class GUIExitPublic(playersNumber: Int) extends RoomExitingRequest with RoomPublicRequest with GUIRequest
 
+  /**
+    * Exits from room view to return a signin view
+    */
+  sealed case class GUIReturnToSignIn() extends GUIRequest
```

Comment:
Questo per separazione di concetti è bene rinominarlo in `case object GUILogOut` spostandolo all'interno di AuthenticationCommon (siccome è un messaggio di auth)

La sua documentazione può essere quella del messaggio LogOut un AuthenticationCommon (senza la doc del parametro)

Preciso che i messaggi li lasciamo entrambi e l'extends lo perdiamo ma fa niente